Bottom: ff6b57b8ce3af09323bea7f9e8ed4c0a1450a339
Top:    9545934a45e4fb1c76a911297a383a953487ef9d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 21:15:27 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 1580d3c..55f8c74 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -820,7 +820,6 @@ ghid_port_window_leave_cb (GtkWidget * widget,
 	}
     }
 
-  CrosshairOff ();
   ghid_show_crosshair (FALSE);
   out->has_entered = FALSE;
